package.json

```diff
@@ -23,7 +23,7 @@
 		"storybook:build": "build-storybook -o docs/public/storybook"
 	},
 	"devDependencies": {
-		"@ag.ds-next/react": "^1.0.0",
+		"@ag.ds-next/react": "^1.3.0",
```

Comment:
Just a note: We will need to upgrade this across the entire monorepo. Also once the avatar fix is released we need to bump this to 1.4.0